unischedule/pubspec.yaml

```diff
@@ -35,6 +35,8 @@ dependencies:
   go_router: ^13.0.0
   flutter_riverpod: ^2.5.1
   local_auth: ^2.2.0
+  multi_dropdown: ^2.1.4
+  flutter_colorpicker: ^1.0.3
```

Comment:
Hey guys, what do you think about these? @SnowArtz @iGotty

Reply:
I think those works perfect for us.

Reply:
These dependency changes seem adequate for our needs. The update to multi_dropdown: ^2.1.4 should give us the new features and bug fixes we've been waiting for.

As for flutter_colorpicker: ^1.0.3, I am familiar with this library and have found it to be quite stable and versatile for our color picking needs.